.github/scripts/get_test_server.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Script that determines the test server to run the tests against.
+
+current_branch=$(git rev-parse --abbrev-ref HEAD)
+echo "current_branch: $current_branch"
+
+if [[ $current_branch = release/* ]]; then
+    sdk_version=$(cat setup.cfg | grep current_version | cut -d ' ' -f 3)
+    echo "sdk_version: $sdk_version"
+    IFS=. read -r major minor patch <<< "$sdk_version"
+    sdk_version_major_minor="$major.$minor"
+
+    prod_version=$(curl --silent --max-time 5 "https://cloud.kili-technology.com/api/label/v2/version" | jq -r .version)
+    echo "prod_version: $prod_version"
+    IFS=. read -r major minor patch <<< "$prod_version"
+    prod_version_major_minor="$major.$minor"
+
+    preprod_version=$(curl --silent --max-time 5 "https://preproduction.cloud.kili-technology.com/api/label/v2/version" | jq -r .version)
+    echo "preprod_version: $preprod_version"
+    IFS=. read -r major minor patch <<< "$preprod_version"
+    preprod_version_major_minor="$major.$minor"
+
+    if [[ $sdk_version_major_minor = $preprod_version_major_minor ]]; then
+        echo "TEST_AGAINST=PREPROD;"
+        echo "KILI_API_ENDPOINT=https://preproduction.cloud.kili-technology.com/api/label/v2/graphql;"
+    elif [[ $sdk_version_major_minor = $prod_version_major_minor ]]; then
+        echo "TEST_AGAINST=PROD;"
+        echo "KILI_API_ENDPOINT=https://cloud.kili-technology.com/api/label/v2/graphql;"
+    else
+        echo "ERROR: SDK version $sdk_version_major_minor does not match neither preprod version $preprod_version nor prod version $prod_version"
+    fi
+else
+    echo "TEST_AGAINST=STAGING;"
+    echo "KILI_API_ENDPOINT=https://staging.cloud.kili-technology.com/api/label/v2/graphql;"
```

Comment:
just a comment: this is not compatible with our wish to decorrelate the SDK release and the app release. This will need to be updated in the coming quarter

Reply:
Yes right. I don't know how to tackle this for now 🤔

Reply:
@theodu why do you think that? Even if we decorrelate the SDK from the app's releases, the SDK will still have to be compatible with all environments. However, if we decorrelate the SDK, we should run E2E tests on LTS environment as well to be sure it remains compatible with old versions from the backend.